src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -105,21 +105,21 @@ public static int LastIndexOfAny(ref byte searchSpace, int searchSpaceLength, re
 
         // Adapted from IndexOf(...)
         [MethodImpl(MethodImplOptions.AggressiveOptimization)]
-        public static unsafe bool Contains(ref byte searchSpace, byte value, int length)
+        public static bool Contains(ref byte searchSpace, byte value, int length)
```

Comment:
There are lots of casts (`(nuint)length`) in this method. (And remember, in 64-bit procs this is a _signed_ extension.) Seems like ideally the 'length' parameter would be taken as a `nuint` instead of an `int` to avoid all of this.

Reply:
Maybe change the Asserts from
```csharp
Debug.Assert(searchSpaceLength >= 0);
Debug.Assert(valueLength >= 0);
```
to 
```csharp
Debug.Assert((nint)searchSpaceLength >= 0);
Debug.Assert((nint)valueLength >= 0);
```
to make it still valid?

Reply:
Changed lengths to `nuint`s